### ISSUE TITLE
Include HL2 tools and weapons as SWEPs

### ISSUE THREAD

**Issue by Kyrasuum** [oochitecht/planetview#9]
Currently in Testing branch PCam is disabled because of a possible workaround that resolves the weapon view mismatch for all SWEPs without the defects of PCam.

HL2 tools and weapons are coded in C so they need to be ported into Lua to receive this fix.

**Comment by Kyrasuum** [comment]
Grav Gun, Pistol, AR2, Smg, and 357 have been made as SWEPs.
Physgun and Toolgun need to be finished before this can be closed.

Currently Grav Gun is using the world model for Physgun but this is a low priority issue.

**Comment by Kyrasuum** [comment]
Toolgun has been Finished.
Physgun will be created using Gravgun as a reference.

Physgun is the only remaining SWEP for this issue.

**Comment by Kyrasuum** [comment]
Most Remaining HL2 SWEPs have been included now.
Physgun still needs coding before this can be closed

**Comment by Kyrasuum** [comment]
This can now be considered resolved.  The physgun is working but is not a final version just yet.
The physgun cannot rotate or change the distance of props as of the current version.
